lxd/storage/drivers/driver_btrfs_utils.go

```diff
@@ -326,9 +327,11 @@ func (d *btrfs) setSubvolumeReadonlyProperty(path string, readonly bool) error {
 // BTRFSSubVolume is the structure used to store information about a subvolume.
 // Note: This is used by both migration and backup subsystems so do not modify without considering both!
 type BTRFSSubVolume struct {
-	Path     string `json:"path" yaml:"path"`         // Path inside the volume where the subvolume belongs (so / is the top of the volume tree).
-	Snapshot string `json:"snapshot" yaml:"snapshot"` // Snapshot name the subvolume belongs to.
-	Readonly bool   `json:"readonly" yaml:"readonly"` // Is the sub volume read only or not.
+	Path         string `json:"path" yaml:"path"`                   // Path inside the volume where the subvolume belongs (so / is the top of the volume tree).
+	Snapshot     string `json:"snapshot" yaml:"snapshot"`           // Snapshot name the subvolume belongs to.
+	Readonly     bool   `json:"readonly" yaml:"readonly"`           // Is the sub volume read only or not.
+	UUID         string `json:"uuid" yaml:"uuid"`                   // The subvolume UUID.
+	ReceivedUUID string `json:"received_uuid" yaml:"received_uuid"` // UUID of the sent volume (when using btrfs receive)
```

Comment:
I'm confused why this field is needed, can you not re-use `UUID` field but in the other direction?

Reply:
I removed it as it wasn't actually used.